profile/README.md

```diff
@@ -1 +1,14 @@
 # Welcome to Open-horizon-services 👋
+<p align="center">
+<img align="center" src="/profile/open-horizon-color.png" height="30%" width="30%"/>
```

Comment:
This won't need be addressed for this review, but we've copied the `open-horizon-color.png` in a lot of places for the logo. I wonder if we should try to load it from the https://github.com/open-horizon/artwork repo instead.